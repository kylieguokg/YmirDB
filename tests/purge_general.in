set a 1 2 3
list entries
snapshot
set b a
list entries
snapshot
set c b
list entries
snapshot
purge a
purge b
purge c
list entries
purge a
purge b
list entries
purge a
list entries
checkout 1
list entries
checkout 2
list entries
checkout 3
list entries
bye
