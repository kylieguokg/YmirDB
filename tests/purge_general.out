> ok

> a [1 2 3]

> saved as snapshot 1

> ok

> b [a]
a [1 2 3]

> saved as snapshot 2

> ok

> c [b]
b [a]
a [1 2 3]

> saved as snapshot 3

> not permitted

> not permitted

> ok

> b [a]
a [1 2 3]

> not permitted

> ok

> a [1 2 3]

> ok

> no entries

> ok

> no entries

> ok

> no entries

> ok

> no entries

> bye
